components/organisms/o-personal-details.vue

```diff
@@ -0,0 +1,258 @@
+<template>
+  <div class="o-personal-details">
+    <SfHeading
+      :title="`1. ${$t('Personal Details')}`"
+      class="sf-heading--left sf-heading--no-underline title"
+    />
+    <div class="form">
+      <SfInput
+        v-model.trim="personalDetails.firstName"
+        class="form__element form__element--half"
+        name="first-name"
+        :label="$t('First name *')"
+        :valid="!$v.personalDetails.firstName.$error"
+        :error-message="
+          !$v.personalDetails.firstName.required
```

Comment:
We don't need to check second condition and this can be typed simpler: `$v.fieldName.requred ? $t('Field is required') : $t('Other message')`

Reply:
Done.